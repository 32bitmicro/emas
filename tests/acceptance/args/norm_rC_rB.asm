lw r0, r0 + r0
lw r0, r0 + r1
lw r0, r0 + r2
lw r0, r0 + r3
lw r0, r0 + r4
lw r0, r0 + r5
lw r0, r0 + r6
lw r0, r0 + r7

lw r0, r1 + r0
lw r0, r1 + r1
lw r0, r1 + r2
lw r0, r1 + r3
lw r0, r1 + r4
lw r0, r1 + r5
lw r0, r1 + r6
lw r0, r1 + r7

lw r0, r7 + r0
lw r0, r7 + r1
lw r0, r7 + r2
lw r0, r7 + r3
lw r0, r7 + r4
lw r0, r7 + r5
lw r0, r7 + r6
lw r0, r7 + r7
