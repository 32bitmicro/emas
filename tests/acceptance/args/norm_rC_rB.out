@ 0x0000 : 0x4000  /  010 000 0 000 000 000  /  16384
@ 0x0001 : 0x4008  /  010 000 0 000 001 000  /  16392
@ 0x0002 : 0x4010  /  010 000 0 000 010 000  /  16400
@ 0x0003 : 0x4018  /  010 000 0 000 011 000  /  16408
@ 0x0004 : 0x4020  /  010 000 0 000 100 000  /  16416
@ 0x0005 : 0x4028  /  010 000 0 000 101 000  /  16424
@ 0x0006 : 0x4030  /  010 000 0 000 110 000  /  16432
@ 0x0007 : 0x4038  /  010 000 0 000 111 000  /  16440
@ 0x0008 : 0x4001  /  010 000 0 000 000 001  /  16385
@ 0x0009 : 0x4009  /  010 000 0 000 001 001  /  16393
@ 0x000a : 0x4011  /  010 000 0 000 010 001  /  16401
@ 0x000b : 0x4019  /  010 000 0 000 011 001  /  16409
@ 0x000c : 0x4021  /  010 000 0 000 100 001  /  16417
@ 0x000d : 0x4029  /  010 000 0 000 101 001  /  16425
@ 0x000e : 0x4031  /  010 000 0 000 110 001  /  16433
@ 0x000f : 0x4039  /  010 000 0 000 111 001  /  16441
@ 0x0010 : 0x4007  /  010 000 0 000 000 111  /  16391
@ 0x0011 : 0x400f  /  010 000 0 000 001 111  /  16399
@ 0x0012 : 0x4017  /  010 000 0 000 010 111  /  16407
@ 0x0013 : 0x401f  /  010 000 0 000 011 111  /  16415
@ 0x0014 : 0x4027  /  010 000 0 000 100 111  /  16423
@ 0x0015 : 0x402f  /  010 000 0 000 101 111  /  16431
@ 0x0016 : 0x4037  /  010 000 0 000 110 111  /  16439
@ 0x0017 : 0x403f  /  010 000 0 000 111 111  /  16447
